worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -52,4 +54,21 @@ To run _Dark Souls III_ in Archipelago mode:
 ### Where do I get a config file?
 
 The [Player Options](/games/Dark%20Souls%20III/player-options) page on the website allows you to
-configure your personal options and export them into a config file.
+configure your personal options and export them into a config file. The [AP client archive] also
+includes an options template.
+
+[AP client archive]: https://github.com/nex3/Dark-Souls-III-Archipelago-client/releases/latest
+
+### Does this work with Proton?
+
+The *Dark Souls III* Archipelago randomizer supports running on Linux under Proton. There are a few
+things to keep in mind:
+
+* Because `DS3Randomizer.exe` relies on the .NET runtime, you'll need to install
+  the [.NET Runtime] under **plain [WINE]**, then run `DS3Randomizer.exe` under
+  plain WINE as well. It won't work as a Proton app!
```

Comment:
Love to see linux instructions! I wonder if they could be a little bit clearer, I still remember starting out on Linux having no clue how to interact with wine myself, relying purely on Proton to do it for me. Might have just been me, thoughts?